### PR TITLE
Features/popular movies api new

### PR DESCRIPTION
**Pull request by RadekBubienko**
Hi guys, I have done:  Api (I added the rest of the queries to the API), main store and saga, slice and saga on PopularMovies and also installed react-redux. I added logic to PopularMovies but something is not working out. It doesn't render. But without error. It downloads the Api but after Loading you can't see anything. I don't know where to look and I'm going through all the code to see if I messed something up. Maybe I'll give a pull-request then together we'll see what's sitting there? Let me know.

### CODE REVIEW

File: src/common/Query/queryParameters.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+export const useQueryParameter = (key) => {
+    const location = useLocation();
+    const searchParams = new URLSearchParams(location.search);
+    
+    return searchParams.get(key);
+};
+
+export const useReplaceQueryParameter = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
```
Comment (konradkapica) [inline src/common/Query/queryParameters.js:11]: shouldn't we use also useHistory hook?

Comment (RadekBubienko) [reply src/common/Query/queryParameters.js:11]: I think so

File: src/features/movies/MovieList/MovieBox/index.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+              <Image
+                src={`https://image.tmdb.org/t/p/original/${movie.poster_path}`}
+                alt=""
+              />
+            ) : (
+              <Image />
+            )}
+            <DescriptionWrapper>
+              {movie.title && <Title>{movie.title}</Title>}
+              {movie.release_date && (movie.department || movie.charakter) ? (
```
Comment (konradkapica) [inline src/features/movies/MovieList/MovieBox/index.js]: shouldn't it be movie.character (with C not K)?

**Review by konradkapica — CHANGES_REQUESTED** [review]
Great job Radek, you put there a lot of work. Can't wait to connect Movie Page and see results :) Only one question in comment in MovieBox/index.js

File: src/features/movies/MovieList/MovieBox/index.js (hunk 2 above)
Comment (RadekBubienko) [reply src/features/movies/MovieList/MovieBox/index.js]: I just corrected.

**Review by konradkapica — APPROVED** [review]

**Pull request merged**